src/System.Net.WebSockets.Client/src/System/Net/WebSockets/WebSocketHandle.Managed.cs

```diff
@@ -204,12 +204,12 @@ public void Abort()
                 }
 
                 // Get or create the buffer to use
-                const int MinBufferSize = 14; // from ManagedWebSocket.MaxMessageHeaderLength
+                const int MinBufferSize = 125; // from ManagedWebSocket.MaxControlPayloadLength
```

Comment:
@davidfowl Does this mean we should pass a 125 byte buffer instead of 14 to `CreateFromStream`?

Reply:
The goal is the remove the buffer from the API completely, so we'll always have a 125 byte buffer.

Reply:
Once this is merged I'll be putting up a separate PR to remove it from the API.